### ISSUE TITLE
Add a max width so it looks okay on wide screens

### ISSUE THREAD

**Issue by nstrayer** [nstrayer/tuftesque2#7]
Right now it gets a bit rough looking on large screens. 

![image](https://user-images.githubusercontent.com/6764693/56852790-17f14c80-68e5-11e9-9cb5-b087941719d4.png)

This is just because I don't have any `max-width` argument. Adding one will make it look much better.

**Comment by Felixmil** [comment]
Agreed ! 
Does max-width should affect side bar though ? I don't think so, right now it has a good enough size. However, the left gutter can be thinner indeed, probably by at least 10%, I'll do some tests and propose differents settings here.

**Comment by Felixmil** [comment]
I increased left-gutter to 12vw and max-width to 1200px, the result is ok on my screen but need to be tested on wider ones.

![image](https://user-images.githubusercontent.com/34234913/56869428-e7063a00-6a00-11e9-90a5-b0ee7fee37a8.png)

**Comment by nstrayer** [comment]
I fixed this in 829927b1f60e34c37931acf1846787fe2c3f9c10 which involved adding another wrapper div that controlled the width of the content + sidebar. This div could then have a max width set on it and be centered. See [these lines](https://github.com/nstrayer/tuftesque2/blob/master/inst/tuftesque.css#L37-L39).
```css
.all_content {
  position: relative;
  max-width: var(--max-page-width) ;
  margin-left: auto;
  margin-right: auto;
}
```
